lib/AST/ASTPrinter.cpp

```diff
@@ -3211,6 +3211,10 @@ static bool usesFeatureTupleConformances(Decl *decl) {
   return false;
 }
 
+static bool usesFeatureSymbolLinkageMarkers(Decl *decl) {
+  return false;
```

Comment:
If we expect the standard library to adopt these new attributes, this will need to be implemented to ensure that we don't condfail (could be a follow up though).

Reply:
Implemented.

Reply:
Thanks! I likely should have elaborated a bit, there may be more to do on this front. Will these new attributes be added to any existing declaration in the standard library or other commonly used modules? If so, this experimental feature has to be declared as a `SUPPRESSIBLE_LANGUAGE_FEATURE`. That way older compilers can pretend the new attributes don't exist and continue parsing those existing decls without behavior changes. On the other hand, if these new attributes will only be used on new declarations then what you have already implemented is probably sufficient since older compilers can probably parse interfaces fine while completely ignoring the existence of the new decls.

Reply:
@tshortli Thanks for the explanation, I get it now.

My intentions are that these attributes are not to be used in the stdlib, and also unlikely to be used/useful in typical framework/modules. But I've heard that @DougGregor has some use case for them too... Doug, do you see these being used in a way that we should protect against condfail?

Reply:
I think I'm going to merge this as is, for now, because the risk seems low: They're underscored attributes guarded via the -experimental flag, so I  think accidental usage is unlikely, and hopefully the stdlib and other common modules are going to think twice before building on top that.

@tshortli are you okay with that?

Reply:
Yes if there aren't immediate plans to use in the stdlib then I'm not worried about what you have now. Thanks!